test/parallel/test-cli-permission-deny-fs.js

```diff
@@ -126,3 +129,19 @@ const fs = require('fs');
   assert.strictEqual(status, 1);
   assert.ok(!fs.existsSync('permission-deny-example.md'));
 }
+
+{
+  const firstPath = path.sep + process.cwd().split(path.sep, 2)[1];
+  const pwd = fixtures.path('permission', 'loader');
+  const file = path.join(pwd, 'index.js');
```

Comment:
I think you should be able to use a single call here: `const file = fixtures.path('permission', 'loader', 'index.js');`